src/Tribe/PUE/Checker.php

```diff
@@ -589,14 +593,15 @@ private function get_api_message( $info ) {
 				'</a>'
 			);
 
-			if ( ! empty( $info->api_invalid_message ) ) {
-				$message = wp_kses( $info->api_invalid_message, 'post' );
+			if ( ! empty( $info->api_inline_invalid_message ) ) {
+				$message = wp_kses( $info->api_inline_invalid_message, 'post' );
 			}
 
 			$message = str_replace( '%plugin_name%', $this->get_plugin_name(), $message );
 			$message = str_replace( '%plugin_slug%', $this->get_slug(), $message );
 			$message = str_replace( '%update_url%', $this->get_pue_update_url(), $message );
 			$message = str_replace( '%version%', $info->version, $message );
+			$message = str_replace( '%changelog%', '<a class="thickbox" title="' . $this->get_plugin_name() . '" href="plugin-install.php?tab=plugin-information&plugin=' . $this->get_slug() . '&TB_iframe=true&width=640&height=808">what\'s new</a>', $message );
```

Comment:
This is old code I restored but you're totally right.

Probably a task for another day but I actually wonder about modifying this system to support something like `%changelog="view the changelog"%` as otherwise:

* When setting these messages the author has to work around the pre-coded wording
* There is also nothing (in the PUE server settings) to remind the author of exactly what `%changelog%` and co are transformed into

Then again, I guess we rarely alter these things.